<!--
   Run the fiducial_slam node
-->
<launch>
  <arg name="mapping_mode" default="true"/>
  <arg name="map_frame" default="map"/>
  <arg name="odom_frame" default="odom"/>
  <arg name="pose_frame" default="base_link"/>
  <arg name="publish_tf" default="true"/>
  <arg name="use_external_pose" default="false"/>
  <arg name="future" default="0.0"/>
  <arg name="median_filter_samples" default="0"/>


<!--
  <node type="fiducial_slam_node.py" pkg="fiducial_slam" output="screen" name="fiducial_slam">
    <param name="map_file" value="$(env HOME)/.ros/slam/map.txt" />
    <param name="obs_file" value="$(env HOME)/.ros/slam/obs.txt" />
    <param name="trans_file" value="$(env HOME)/.ros/slam/trans.txt" />
    <param name="mapping_mode" value="$(arg mapping_mode)" />
    <param name="map_frame" value="$(arg map_frame)" />
    <param name="odom_frame" value="$(arg odom_frame)" />
    <param name="pose_frame" value="$(arg pose_frame)" />
    <param name="publish_tf" value="$(arg publish_tf)" />
    <param name="future" value="$(arg future)" />
    <param name="use_external_pose" value="$(arg use_external_pose)" />
    <param name="median_filter_samples" value="$(arg median_filter_samples)" />
  </node>
-->
</launch>
